src/components/PlayerControls.tsx

```diff
@@ -45,20 +45,20 @@ const PlayerControls: FunctionComponent<PlayerControlsProps> = ({
             )}
           </div>
           <div className="flex items-center">
-            {/* <ListboxInput value={'' + speed} onChange={setSpeed}>
+            <ListboxInput value={speed} onChange={setSpeed}>
               <ListboxButton className="w-20 bg-gray-300 rounded">
                 <IconChevronRight className="w-4" /> x{speed}
               </ListboxButton>
               <ListboxPopover>
                 <ListboxList>
                   {availableSpeeds.map((option) => (
-                    <ListboxOption key={option} value={'' + option}>
+                    <ListboxOption key={option} value={option}>
```

Comment:
this appears to work as expected

I think we are going to move this control back into the player. Talking to @vojtaholik about it.